src/Revise.jl

```diff
@@ -301,10 +330,29 @@ Watch `file` for updates and [`revise`](@ref) loaded code with any
 changes. `mod` is the module into which `file` is evaluated; if omitted,
 it defaults to `Main`.
 """
-function track(mod::Module, file::AbstractString)
+function track(mod::Module, file::AbstractString; reference::Symbol=:current)
```

Comment:
Would it be better to put the `Compiler` part into the method for `track(::Module)`? This one is really intended for module-scoping rather than tracking all the code that defines a module. (In reality this is probably left-over legacy from the bad old days when Revise had to parse the source code and try to figure out which files were `include`d into the module. Not fully certain of that, though.)

Reply:
I figured the generic "track from git' functionality might be useful in other situations, hence the split. But I'm not happy with the quick and dirty interface though.